src/ui/window.rs

```diff
@@ -90,11 +90,13 @@ impl<D> WindowBuilder<D> {
         self
     }
 
+    #[allow(dead_code)]
```

Comment:
why do you want to keep them?

Reply:
this is an API of UI widgets. I have plans for them

Reply:
Okay, gotcha